gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -376,7 +376,6 @@ sub ref!RenderPassObject createRenderPassObjectFromInfo2(
       StencilStoreOp: attachment.stencilStoreOp,
       InitialLayout: attachment.initialLayout,
       FinalLayout: attachment.finalLayout,
-      StencilLayout: null,
```

Comment:
I would keep the null if possible to explicitly show that it's not set.

While looking at the constructor, it's not obvious to see there is an unset field.

Reply:
Omitting null members seems to be a pretty common pattern in the api files, but I'm happy to put it back in